roles/install-app/tasks/main.yml

```diff
@@ -101,3 +110,26 @@
     - install
     - app
     - docs
+
+#TODO use ansible_selinux
+- name: Permit nginx to connect to backend
+  when: ansible_distribution == "CentOS"
+  seboolean:
+    name: httpd_can_network_connect
+    state: yes
+    persistent: yes
+  tags:
+    - install
+    - app
+
+# the optionnal yaml.Cloader is never installed by pip
+# This hack install it in virtualenv
```

Comment:
On my debian I got the same problem, yaml.Cloader is never compiled in the virtualenv and I have a patch for copying it the same way